### DISCUSSION TITLE
The Changes Does not Reflect on The Site

### DISCUSSION THREAD

**Discussion by aldoprogrammer** [community/community#58742] [New to GitHub] [answered]
I don't know what went wrong, but I already rebuild my web app, also already commit the changes that I made, you can check on my repository here: https://github.com/aldoprogrammer/umb-chatbot

And the live site still does not reflect the changes, https://github.com/aldoprogrammer/umb-chatbot

**Comment by anhtran917** [comment]
Hello, I recommend you make sure some things for your project.
1. Sometimes, it can take a few minutes for changes to your website to appear on the live site. 
   Try waiting a few minutes and refreshing the page to see if the changes have appeared.

2. Check the build status: When you push changes to your repository, GitHub Pages will automatically build your website and publish the changes. You can check the build status of your website by going to the "Settings" tab of your repository and scrolling down to the "GitHub Pages" section. If there are any errors or warnings in the build process, they will be displayed here.

3. Check the branch: Make sure that you are pushing your changes to the correct branch of your repository. By default, GitHub Pages will use the "master" branch to build your website. If you are using a different branch, make sure that you have specified the correct branch in the "GitHub Pages" section of your repository settings.

I hope this will be helpful for you. Regards, 
Anh

**Comment by dhrubasaha08** [comment]
Hello, @aldoprogrammer It appears that you're facing an issue where the changes you made to your web app and committed to your repository aren't being reflected on the live site. To troubleshoot this problem, you can try the following steps:
- Verify the deployment process: Make sure that you have correctly deployed your web app after committing the changes. Check if there are any specific build or deployment steps required for your app, such as running a build script or configuring deployment settings.
- Confirm the deployment target: Ensure that you are deploying to the correct environment or hosting service. Check if the deployment settings in your repository match the hosting platform you are using for your live site.
- Double-check the branch and commit: Ensure that you are deploying the changes from the correct branch and that the commit you made includes the desired modifications. It's possible that the changes were made on a different branch or the commit was not properly pushed.
- Review the deployment logs: Check the deployment logs or any error messages provided by the hosting service. They can provide insights into any issues encountered during the deployment process.
- Clear browser cache: Sometimes, the changes might not be immediately visible due to browser caching. Try clearing your browser cache or accessing the live site using a different browser or in incognito mode to see if the changes are reflected.

**Comment by aldoprogrammer** [comment, marked as answer]
I found the problem and solution, so that the website itself it still accessing the cache of the old version, so all I did is to `remove the branch gh-pages from VS Code`, make sure that you already change the branch to master so that you are allowed to remove gh-pages branch, then go to VS Code, start rebuild it again using `npm run build`, after that push the changes. go to the settings, and change it again from `master branch` to `gh-pages` again, then the changes reflected.

By using this methode, so that the website will not access the website cache which if it does, it still showing the old version even though we already push the changes on github repository.
